framework-profile.yaml

```diff
@@ -2,14 +2,16 @@ common:
   - system/base-cloud-config
   - dracut/immutable-rootfs
   - dracut/kcrypt
-  - dracut/network
```

Comment:
this was an oversight, we can't use it as is by default, as systemd-networkd support is not working enough yet with netboot